### DISCUSSION TITLE
Welcome to Hactoberfest-2022-New Discussions!

### DISCUSSION THREAD

**Discussion by prashantkalokhe** [prashantkalokhe/Hactoberfest-2022-New#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prashantkalokhe** [comment]
Pull request are merged 4 to 5
Time day just make Pull request I will merge it
All Pull request are to merged , don't take tensions.

**Reply by HarshAggarwal1** [reply]
Hacktoberfest excluded this repo.

**Comment by HarshAggarwal1** [comment]
Why this repository is excluded from hacktoberfest 2022. Hacktoberfest not anymore accepting this repository.
